packages/muelu/doc/UsersGuide/masterList.xml

```diff
@@ -411,6 +411,24 @@
       <comment-ML>parameter not existing in ML</comment-ML>
     </parameter>
 
+    <parameter>
+      <name>aggregation: classical scheme</name>
+      <type>string</type>
+      <default>"direct"</default>
+      <description>Prolongator formation option for classical coarsening.</description>
+      <visible>false</visible>
+      <comment-ML>parameter not existing in ML</comment-ML>
+    </parameter>
+
+    <parameter>
+      <name>aggregation: row sum drop tol</name>
```

Comment:
Remove `refmaxwell: row sum drop tol`.

Reply:
?

Reply:
I think we still need those guys for stuff at the top level.